### PR TITLE
Add CakePHP 3 recipe

### PR DESCRIPTION
**Pull request by jadb**
Would be nice to have this recipe added as an example to the CakePHP folks using the library.

Great work by the way!

### CODE REVIEW

File: recipe/cakephp.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * file that was distributed with this source code.
+ */
+
+require_once __DIR__ . '/common.php';
+
+/**
+ * CakePHP 3 Project Template configuration
+ */
+
+// Yii 2 Advanced Project Template shared dirs
```
Comment (ceeram) [inline recipe/cakephp.php]: wat?

Comment (jadb) [inline recipe/cakephp.php]: I would have sworn I had changed all my copy/pasting...

**Comment by ADmad** [comment]
There should be a task to symlink plugin asset into webroot too, using `bin/cake plugin assets symlink`.

File: recipe/cakephp.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+set('shared_files', [
+    'config/app.php',
+]);
+
+/**
+ * Initialization
+ */
+task('deploy:init', function () {
+    run('{{release_path}}/bin/cake orm_cache clear');
+    run('{{release_path}}/bin/cake orm_cache build');
```
Comment (lorenzo) [inline recipe/cakephp.php]: This should be run after the migrations

Comment (elfet) [inline recipe/cakephp.php]: @jadb please, fix it. And i will merge it.

**Comment by elfet** [comment]
Great! I'll merge. @deployphp/maintainer ?

**Comment by jadb** [comment]
Thanks! Were you pinging the rest of the team or inviting me to join the maintainers (to take care of this one only)? Sorry if I misunderstood and thanks again for merging!

**Comment by elfet** [comment]
Any contributions are welcome

**Pull request merged**